core/vocabulary/src/main/java/org/trellisldp/vocabulary/ACL.java

```diff
@@ -25,31 +25,38 @@
  * @author acoburn
  */
 public final class ACL {
+    private static final String NS = "http://www.w3.org/ns/auth/acl#";
```

Comment:
If you change `NS` for `URI` (effectively, making `URI` a private field), you'll make this PR much smaller. Clearly, for the `DC` class, you'd need a value such as `NS`, but for most classes the `URI` field won't conflict.